tests/src/system/basic/WskBasicTests.scala

```diff
@@ -295,6 +295,17 @@ class WskBasicTests
                 .stderr should include regex (""""error": "This error thrown on purpose by the action."""")
     }
 
+    it should "create and invoke a blocking action resulting in an error response object" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "errorResponseObject"
+            assetHelper.withCleaner(wsk.action, name) {
+                (action, _) => action.create(name, Some(TestUtils.getTestActionFilename("errorObject.js")))
+            }
+
+            wsk.action.invoke(name, blocking = true, expectedExitCode = 246)
```

Comment:
is it the case that for a blocking invoke where the action returns an error response, the CLI was not interpreting the response correctly? if so, the use of whisk.invoke is a red herring. I would simplify the test to use an action that just returns a failed promise.